x-pack/plugins/security_solution/server/endpoint/mocks.ts

```diff
@@ -90,6 +93,19 @@ export const createMockPackageService = (): jest.Mocked<PackageService> => {
   };
 };
 
+/**
+ * Create mock AgentPolicyService
+ */
+
+export const createMockAgentPolicyService = (): jest.Mocked<AgentPolicyService> => {
```

Comment:
this function should be in Fleet and exposed via this `/server` namespace. we should then import it from there

Reply:
I'll move the agent service mock over as well